README.md

```diff
@@ -1,2 +1,33 @@
-# pykla-web
-The official website for Python Kampla (PyKla)
+# Pykla-web
+
+The official website for Python Kampala (PyKla)
+
+## Technologies
+
+* Python 3.6
+* SQLite3
+
+## Requirements
+
+* Install [Python](https://www.python.org/downloads/)
```

Comment:
You could Add what you've put under a Windows section. Someone else will add a Unix sepcific section later.

Reply:
Hello @lubegamark, thank you for the feedback. I think I will just add details for Mac OS.